content/http-handler-abi.md

```diff
@@ -859,6 +859,31 @@ below, and the `len` would be eight.
           buf --^
 ```
 
+### `get_remote_addr`
+
+```webassembly
```

Comment:
since the spec name is "http-wasm" and also all the existing ABIs are related to http req/resp, what is the rationale for adding a new abi which is for tcp layer details?

Reply:
well, Remote address aka source address is pretty important thing when you do something with http. Lets say: you want to do http middleware that will for instance block requests based on countries. You simply cannot do that if you cannot see the client source address.

Reply:
Also: source address is must have if you want to do like audit logging, without that the audit log is useless.

Reply:
make sense. Thanks